vector/src/test/java/im/vector/app/features/settings/devices/v2/DevicesViewModelTest.kt

```diff
@@ -57,6 +63,17 @@ class DevicesViewModelTest {
         )
     }
 
+    @Before
```

Comment:
Do we really need this mock on `SystemClock`?

Reply:
It is somehow failing on my machine. I didn't see a harm to mock it.